### ISSUE TITLE
ignore W7930 for XML files not to be handled by manifest

### ISSUE THREAD

**Issue by eLBati** [OCA/pylint-odoo#102]
See
https://travis-ci.org/OCA/l10n-italy/jobs/190682978
in particular
```l10n_it_fatturapa_out/tests/data/IT06363391001_00001.xml:1: [W7930(file-not-used), ]  not used from manifest```

That XML file is used by automatic tests, to check the result of invoice export, so it must not be used by manifest.
So, pylint-odoo should not raise the error.

**Comment by pedrobaeza** [comment]
It's only a warning, so no special problem. Anyway, there's some excluded folder that you can use for holding this kind of files without even warning. I don't remember now which. @moylop260, do you?

**Comment by eLBati** [comment]
But https://github.com/OCA/l10n-italy/pull/265 is red because of lint checks
Isn't it because of W7930?
Thanks

**Comment by pedrobaeza** [comment]
No, it's red because of:
```
l10n_it_fatturapa/bindings/_ds.py:10: [W7935(missing-import-error), ] External dependency "pyxb" without ImportError. More info: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#external-dependencies

l10n_it_fatturapa/bindings/_ds.py:11: [W7935(missing-import-error), ] External dependency "pyxb.binding" without ImportError. More info: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#external-dependencies

l10n_it_fatturapa/bindings/_ds.py:12: [W7935(missing-import-error), ] External dependency "pyxb.binding.saxer" without ImportError. More info: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#external-dependencies

l10n_it_fatturapa/bindings/_ds.py:14: [W7935(missing-import-error), ] External dependency "pyxb.utils.utility" without ImportError. More info: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#external-dependencies

l10n_it_fatturapa/bindings/_ds.py:15: [W7935(missing-import-error), ] External dependency "pyxb.utils.domutils" without ImportError. More info: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#external-dependencies

l10n_it_fatturapa/bindings/_ds.py:16: [W7935(missing-import-error), ] External dependency "pyxb.utils.six" without ImportError. More info: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#external-dependencies

l10n_it_fatturapa/bindings/_ds.py:19: [W7935(missing-import-error), ] External dependency "pyxb.binding.datatypes" without ImportError. More info: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#external-dependencies

************* Module l10n_it_fatturapa.bindings.fatturapa_v_1_2

l10n_it_fatturapa/bindings/fatturapa_v_1_2.py:10: [W7935(missing-import-error), ] External dependency "pyxb" without ImportError. More info: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#external-dependencies

l10n_it_fatturapa/bindings/fatturapa_v_1_2.py:11: [W7935(missing-import-error), ] External dependency "pyxb.binding" without ImportError. More info: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#external-dependencies

l10n_it_fatturapa/bindings/fatturapa_v_1_2.py:12: [W7935(missing-import-error), ] External dependency "pyxb.binding.saxer" without ImportError. More info: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#external-dependencies

l10n_it_fatturapa/bindings/fatturapa_v_1_2.py:14: [W7935(missing-import-error), ] External dependency "pyxb.utils.utility" without ImportError. More info: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#external-dependencies

l10n_it_fatturapa/bindings/fatturapa_v_1_2.py:15: [W7935(missing-import-error), ] External dependency "pyxb.utils.domutils" without ImportError. More info: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#external-dependencies

l10n_it_fatturapa/bindings/fatturapa_v_1_2.py:16: [W7935(missing-import-error), ] External dependency "pyxb.utils.six" without ImportError. More info: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#external-dependencies

l10n_it_fatturapa/bindings/fatturapa_v_1_2.py:19: [W7935(missing-import-error), ] External dependency "pyxb.binding.datatypes" without ImportError. More info: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#external-dependencies

l10n_it_fatturapa/bindings/fatturapa_v_1_2.py:20: [W0403(relative-import), ] Relative import '_ds', should be 'l10n_it_fatturapa.bindings._ds'

************* Module l10n_it_fatturapa_out.wizard.wizard_export_fatturapa

l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py:7: [W7935(missing-import-error), ] External dependency "unidecode" without ImportError. More info: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#external-dependencies

l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py:9: [W7935(missing-import-error), ] External dependency "pyxb.exceptions_" without ImportError. More info: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#external-dependencies

l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py:9: [W7936(missing-manifest-dependency), ] Missing external dependency "pyxb.exceptions_" from manifest. More info: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#external-dependencies
```

**Comment by eLBati** [comment]
@pedrobaeza ok, thanks. PR is green now.

Let's just see if @moylop260 knows the excluded folder

**Comment by moylop260** [comment]
I tried add this case in the README of this project
I'm sorry if It isn't clear.
Any suggestion is welcome.

https://github.com/OCA/pylint-odoo#usage
`If you have external files you can add them in examples folder to skip.`

**Comment by moylop260** [comment]
Maybe we should consider tests folder too like a folder to skip this check.

**Comment by pedrobaeza** [comment]
Yes, `tests` is a good candidate also.

**Comment by moylop260** [comment]
Tests folders skipped from https://github.com/OCA/pylint-odoo/pull/114
